playbooks/provisioning/openstack/README.md

```diff
@@ -325,7 +325,23 @@ if requested, and DNS server, and ensures other OpenShift requirements to be met
 
 ### Running Custom Post-Provision Actions
 
-If you'd like to run post-provision actions, you can do so by creating a custom playbook. Here's one example that adds additional YUM repositories:
+A library of custom post-provision actions exists in `openshift-ansible-contrib/playbooks/provision/openstack/custom-actions`. Playbooks include:
+
+  - add-yum-repos.yml
```

Comment:
Could you add a short description of what it does? Something like "It adds a list of custom yum repositories to every node in the cluster".

What do you think about creating an new subheading for every custom action? So people skimming the doc can notice it more easily? And it would be nice to create a hyperlink to the playbook as well, so you can click on it here and see its contents.

Reply:
Makes sense! I'll do it.